ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -581,7 +581,122 @@ FCU_DFSS_DFVSC_ZTC_CHWZTC:
   - ZTC
   - CHWZTC
 
+FCU_DFSS_DFVSC_ZTC_ZHC_CHWDC_FDPM_DMM:
+  description: "Hydronic varaible speed fan coil with cooling and heating (zone temp control)."
```

Comment:
the next 6 descriptions in this change are the same.  Please differentiate.

Reply:
adjusted.